### PR TITLE
QOL improvements - 96.7a 7cav 0.1 

### LINKED ISSUE #4: Investigate options related to kp player menu and people not squadding
https://7cav.us/threads/4975/#post-26961

### LINKED ISSUE #5: Investigate options regarding HALO jumping
https://7cav.us/threads/4975/#post-26961

Potentially disable when Pilot is online or just disable entirely

### LINKED ISSUE #6: Disable ACE blueforce tracker
we have ctab for this, no need for ace BFT on map

https://7cav.us/threads/4975/#post-26961

### LINKED ISSUE #7: Add MP slots and Mission Controller slots
Also tie these slots in to the zeus GUIDs

### PR DESCRIPTION
**Pull request by uaon**
Quality of Life:
- Added new vehicles for blufor preset.
- Added HEAL script addaction for FOB/Mobile FOB
- Removed check for recycle or not on supply containers.
- Added (Modified) Advanced Towing script.
- Changed civilian behavior to actually run when in danger to nearest building.
- Added more buildable assets for FOB
- Customized cba_settings.sqf to fit the mission format.
- Removed AH-64D from having to be unlocked.
- Added FIR A10C as a optional airframe (Needs unlock)
- Added CWIS AA as a optional base defense.
- Changed default parameters to be more inline with production.
- Added whitelist for most of S3/S6 Staff working or needing access to commander or zeus.
- Adjusted costs of standard HMMWV to allow for easier access to early vehicles.
- Adjusted Reputation costs to be harder but more rewards for doing taks.
- Adjusted Intel to reduce odd numbers when accumulating intel. (being 1 digit shy of next goal)
- Replaced Tent Hanger (AIR CONTROLLER) with Diesel Ground Power unit due to FIRAWS believing its a resupply point for presets.
- added check for pilots, to disable HALO Jumping if one exists.
- added mp whitelist for spectator cam
- change intro music to something less ear rape.
- fixed grouping issue on slot joining where it would isolate you into seperate groups regardless of selection in lobby.

Optimizations:
- replaced ``==`` and ``!=`` statements with EqualTo and NotEqualTo for faster performance.
- added Event handlers for ifFired to stop using depreciated wait until loops.
- fixed EH on drop rope by deleting the detach object before ending the add action. (ADV towing)

### CODE REVIEW

File: Missionframework/cba_settings.sqf [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // ACE Advanced Ballistics
-force force ace_advanced_ballistics_ammoTemperatureEnabled = false;
-force force ace_advanced_ballistics_barrelLengthInfluenceEnabled = false;
+force force ace_advanced_ballistics_ammoTemperatureEnabled = true;
+force force ace_advanced_ballistics_barrelLengthInfluenceEnabled = true;
 force force ace_advanced_ballistics_bulletTraceEnabled = false;
-force force ace_advanced_ballistics_enabled = false;
+force force ace_advanced_ballistics_enabled = true;
```
Comment (NikPoliakov) [inline Missionframework/cba_settings.sqf:5]: LOL NO

**Comment by uaon** [comment]
merged from develop with new changes.

**Comment by uaon** [comment]
Ready for Review and Deployment, Tests pass on latest : commit  #3

**Review by NikPoliakov — APPROVED** [review]
10/10 will merge again

**Pull request merged**